### ISSUE TITLE
Several atmosphere variable issues in KSP 1.10.0

### ISSUE THREAD

**Issue by Neo2323** [JonnyOThan/RasterPropMonitor#17]
Hi, after some tests with the recent version and KSP 1.10.0, most stuff seems to be working correctly, but I found some problems with atmospheric data using ASET and ASET based props. 
Atmosphere variables store the wrong values. For example the ATM gauge shows 91% on launch pad (it was 99% before), 20% at 10000 m and it reaches zero at 20000 m. The stock atmosphere  gauge is just under 50% at this point. Stock and RPM values do not match and are way off. 
Respectively the "Atmosphere Density" and the "Atmosphere Pressure" digital readouts are not correct
too. 
I am a huge IVA fan and made over 70 complex IVA for stock and mods, so RPM is one of my must have mods. I hope this can be fixed. Thank you for your time and keep up the good work!

**Comment by JonnyOThan** [comment]
Darn, I probably screwed something up when moving the data off the stock UI.  It would be useful if you could provide the specific RPM variable names being used if you know them.  If not that, then the prop names.  If not that, then a screenshot and the name of the pod/IVA mod pack.

**Comment by Neo2323** [comment]
Thanks for the response. The three variables in question are: ATMOSPHEREDEPTH (for example used by: IndicatorADV_ATMDEPTH), ATMDENSITY (for example used by: DigitalIndicator_AtmDen) and ATMPRESSURE (for example used by: DigitalIndicator_ATMPRESS).
I think I got confused or remembered the values different. Density and pressure variables seem to be ok after all. At least they did not change much. I have made a test with two KSP/RPM versions, launched from the KSC on Kerbin:

KSP 1.7.1
Launch Pad: Density 1.118 - Pressure 0.987 - ATM 99%
At 10.000 m: Density 0.278 - Pressure 0.175 - ATM 69%
At 20.000 m: Density 0.036 - Pressure 0.023 - ATM 41%
At 30.000 m: Density 0.006 - Pressure 0.004 - ATM 26%
At 45.000 m: Density 0.000 - Pressure 0.000 - ATM 14%

KSP 1.10.0
Launch Pad: Density 1.144 - Pressure 0.988 - ATM 92%
At 10.000 m: Density 0.280 - Pressure 0.175 - ATM 23%
At 20.000 m: Density 0.038 - Pressure 0.024 - ATM 2%
At 30.000 m: Density 0.006 - Pressure 0.004 - ATM 0%
At 45.000 m: Density 0.000 - Pressure 0.000 - ATM 0%

I am glad you already found the ATM problem. The other two variables are very similar in both test flights, so they where probably a false alarm. Sorry about that. I find it a bit strange though. I can aerobrake or reenter in an atmosphere without density and zero pressure while it happens. Sounds pretty kerbal to me...

I often thought about releasing my work, but at this time there are no public downloads available. I used many assets from other sources or mods to make custom props. I included many blender modified interior meshes and/or edited command pods from other mods to my collection and made them fit. I have no knowledge about all those copyright and licence stuff, so that is a problem. On some things, I don't even remember who the original author was over the years. But maybe someday I will make my IVA-Packs public somehow.

Thank you again!
